### ISSUE TITLE
Utxo service returning status code 400 during transfer

### ISSUE THREAD

**Issue by satran004** [bloxbean/cardano-client-lib#26]
Utxo service throws 400 status code with the following error

"Unable to get enough Utxos for address : addr_test1qzx9hu8j4ah3auytk0mwcupd69hpc52t0cw39a65ndrah86djs784u92a3m5w475w3w35tyd6v3qumkze80j8a6h5tuqq5xe8y, reason: {"status_code":400,"error":"Bad Request","message":"querystring.page should be >= 1"}"

Root Cause:
It seems Blockfrost api for utxos now takes page no >= 1. But currently cardano-client-lib starts with page no = 0.

Changes required in DefaultUtxoSelectionStrategy class to set the start index to 1.

**Comment by satran004** [comment]
@JustinWYL, you are right. Probably, it makes sense to add another overloaded method in BackendService for getFeeCalculationService to provide an instance of TransactionHelperService.

```
backendService.getFeeCalculationService(TransactionHelperService)
```

**Comment by nemo83** [comment]
IS this fixed in 01.4? I some how.. .still have it?!

**Comment by satran004** [comment]
Fixed in 0.1.4. Pagination starts from 1
